apps/playground/src/app/components/indicators/label/label.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, Input } from '@angular/core';
+import { SkyIndicatorDescriptionType, SkyLabelType } from '@skyux/indicators';
+
+@Component({
+  selector: 'app-label',
+  templateUrl: './label.component.html',
+})
+export class LabelDemoComponent {
+  @Input()
```

Comment:
I know this is the playground - but is any of this code even needed or is this just copied from the code example and not modified? It appears that the HTML file isn't actually using any of this.